contracts/contracts/BidderRegistry.sol

```diff
@@ -240,12 +250,10 @@ contract BidderRegistry is
             totalAmount += amount;
         }
 
-        if (totalAmount == 0) {
```

Comment:
was it intentional to remove this?

Reply:
Yes, bcs this check was happening on each function call, it's cheaper to remove it.